services/src/main/java/org/exoplatform/news/storage/jcr/JcrNewsStorage.java

```diff
@@ -1267,5 +1271,18 @@ private Session getSession(SessionProvider sessionProvider) throws RepositoryExc
                                                        .getDefaultWorkspaceName(),
                                       repositoryService.getCurrentRepository());
   }
-  
+
+  private  void updateNewsImagesPermissions(News news ,  SessionProvider sessionProvider, Space space  ) throws RepositoryException {
+    Matcher matcher = IMAGE_SRC_PATTERN.matcher(news.getBody());
+    while (matcher.find()) {
+      String match = matcher.group(1);
+      String imageUUID = match.substring(match.lastIndexOf("/") + 1);
+      ExtendedNode image = (ExtendedNode) getNodeById(imageUUID, sessionProvider);
+      if (image.canAddMixin(EXO_PRIVILEGEABLE)) {
+        image.addMixin(EXO_PRIVILEGEABLE);
+      }
```

Comment:
Is there a need to add this mixin ? you already loaded the image by UUID ?

Reply:
we could not set the permission without adding EXO_PRIVILEGEABLE

Reply:
Yes you are right, I misread it